ntbs-service/Pages/Notifications/Create.cshtml.cs

```diff
@@ -20,7 +20,10 @@ public async Task<IActionResult> OnGetAsync()
             var notification = new Notification();
             notification.PatientDetails = new PatientDetails();
             notification.Episode = new Episode();
```

Comment:
Ok so i noticed we weren't consistent here, and it led to issues later on when I made some changes in how to do updates (see comments in NotificationService)- I think it makes sense to initialise everything in here regardless.

Reply:
Yes, it definitely does!